openwisp_controller/config/base/device.py

```diff
@@ -141,6 +141,25 @@ def generate_key(self, shared_secret):
         else:
             return KeyField.default_callable()
 
+    def _assert_org_not_null(self):
+        if not hasattr(self, 'organization'):
+            raise ValidationError({'organization': _('This field cannot be null.')})
```

Comment:
are you sure this is necessary? It should be done by django already.
If you're doing this because otherwise `_assert_name_unique` would fail, you can simply skip the check if organization is `None`, django should take care of it and we can avoid redundancy